src/classes/App.js

```diff
@@ -6,6 +6,15 @@ import '../styles/App.css';
 import '../styles/human.css';
 
 class App extends Component {
+
+    componentWillMount(){
+        document.body.dataset.standalone = process.env.REACT_APP_STANDALONE;
+    }
+    
+    componentWillUnmount(){
```

Comment:
I think we can remove this as this component should never unmount except for when we close the page, at which point the value of the body dataset does not matter.